test/cpp/jit/test_lite_interpreter.cpp

```diff
@@ -337,6 +338,29 @@ class TorchBindLiteInterpreterTestStruct
   }
 };
 
+TEST(LiteInterpreterTest, ToBackend) {
+  Module m("M");
+  m.define(R"JIT(
+    def forward(self, x):
+      return 2 * x
+  )JIT");
+
+  c10::Dict<IValue, IValue> compile_spec(StringType::get(), AnyType::get());
+  c10::Dict<IValue, IValue> fake_dict(StringType::get(), AnyType::get());
+  fake_dict.insert("", "");
+  compile_spec.insert("forward", fake_dict);
+  auto any_dict_ty = DictType::create(StringType::get(), AnyType::get());
+  auto lowered_m = torch::jit::detail::codegen_backend_module(
+      "test_backend", m, compile_spec, any_dict_ty);
+//  lowered_m.save("/Users/myuan/models/backend/backend_c.pt");
+
+  auto minput = 5 * torch::ones({});
+  auto ref = m.run_method("forward", minput).toTensor().item<float>();
+  auto res = lowered_m.run_method("forward", minput);
```

Comment:
I got error from this line:
C++ exception with description "The following operation failed in the TorchScript interpreter.
Traceback of TorchScript (most recent call last):
RuntimeError: Expected 1 elements in a list but found 2
" thrown in the test body.
@SplitInfinity , anything I did wrong?

Reply:
I think this exception is caused by the list unpack operator generated by this line of code inside `codegen_backend_module`:
```
$unpack, = self.__backend.execute(self.__handles["$method"], typed_inputs)
```

I noticed that you used the `test_backend` when calling `codegen_backend_module`. The test backend that we use to test this feature is very basic and is hardcoded to return two outputs for `forward`. You can see the implementation in `test/cpp/jit/test_backend.cpp` (relative to root). I would recommend implementing your own backend that is more intelligent or implements exactly the ops you are testing.

Reply:
Aha! That makes sense. I'll implement the backend.